docs/style-guide.md

```diff
@@ -199,3 +201,22 @@ We could enforce a camel case convention on these strings, such as the following
 ##### Decision
 
 - Defer the decision on the syntax for identifiers to the other specification when possible.
+
+## Element Ordering
+
+This section concerns the order in which containers store elements. 
+
+### General Guidelines
+
+:star2: ECMA-402 spec must provide a deterministic order for the contents of all containers. This order should be lexicographic except in cases wherein there is a clearly preferable semantic ordering. :star2:
```

Comment:
```suggestion
:star2: *ECMA-402 spec must provide a deterministic order for the contents of all containers. This order should be lexicographic except in cases wherein there is a clearly preferable semantic ordering.* :star2:
```

Reply:
Consistency with existing conventions at [Identifiers defined by ECMA-402](https://github.com/tc39/ecma402/blob/ff6c9c68d629d1cdc3edf45755262e15d17e84b1/docs/style-guide.md#identifiers-defined-by-ecma-402) and [Identifiers defined outside ECMA-402](https://github.com/tc39/ecma402/blob/ff6c9c68d629d1cdc3edf45755262e15d17e84b1/docs/style-guide.md#identifiers-defined-outside-ecma-402).